integration-cli/docker_cli_start_volume_driver_unix_test.go

```diff
@@ -119,11 +119,6 @@ func (s *DockerExternalVolumeSuite) SetUpSuite(c *check.C) {
 			http.Error(w, err.Error(), 500)
 		}
 
-		p := hostVolumePath(pr.name)
-		if err := os.RemoveAll(p); err != nil {
-			http.Error(w, err.Error(), 500)
-		}
-
 		w.Header().Set("Content-Type", "application/vnd.docker.plugins.v1+json")
```

Comment:
Looks like you're not bumping the API version, but you are changing the API? This will make it difficult for plugin authors to implement the right interface.

It would be great if this PR could bump the plugin API version number and somehow support compatibility for plugin implementations that implement the older v1 API (although I can't see immediately how to do this). If that's not possible, then just bumping the API version would be an improvement because plugins can then selectively provide different interfaces depending on the version sent.